### ISSUE TITLE
XDR copy not working

### ISSUE THREAD

**Issue by Garvit-k** [script3/yieldblox-beta#67]
When there's a transaction failure the prompt shows XDR copy option but clicking on it doesn't copy the XDR and closes the prompt

**Comment by vv181** [comment]
@Garvit-k 

![img](https://i.imgur.com/ZOtvGuF.png)

Do you mean that? 

Even if I got "'<' is an invalid start of a value. Path: $ | LineNumber: 0 | BytePositionInLine: 0."", It works fine on my end. I using Firefox, and able to copy the XDR.

**Comment by Garvit-k** [comment]
yes this same error but copy operation didnt work for me i am using firfox developer edition maybe this is an browser clipboard API inconsistency

**Comment by drew-boy** [comment]
Hello @Garvit-k!

Could you comment one of these on your issue:

Discord username
Keybase username
Email linked to your beta account
Your YieldBlox Testnet Beta public address
Your personal public Stellar address (not recommended; post at your own risk)

It's so we can contact you and reward you a Beta Bug Bounty! :-)

**Comment by Garvit-k** [comment]
Discord username : Remloy#338
Keybase username : garvit
Email linked to your beta account : garvitkothari1@gmail.com
Your YieldBlox Testnet Beta public address : GD772AGRSRJ5KUFTPT6MCWKS2EC2OKWLT77HDOA2Y74HFOPAUIDM24GI
Your personal public Stellar address (not recommended; post at your own risk) : can this be collected via discord DM ?

**Comment by markuspluna** [comment]
Bug Bounty paid in transaction:

https://laboratory.stellar.org/#explorer?resource=transactions&endpoint=single&values=eyJ0cmFuc2FjdGlvbiI6IjJkNzVjZTM1YTdiYTdhOTViYzczMjk1MTE2M2RlODQ0MWQ0NzY5Y2Q3OGY3NTY1ODg3MmFmZWZkZjdhNTVlMzAifQ%3D%3D&network=public
